modules/tinymce/src/themes/silver/main/ts/backstage/Backstage.ts

```diff
@@ -58,7 +59,27 @@ const init = (lazySinks: { popup: () => Result<AlloyComponent, string>; dialog:
     translate: I18n.translate,
     isDisabled: () => editor.mode.isReadOnly() || !editor.ui.isEnabled(),
     getOption: editor.options.get,
-    tooltips: TooltipsBackstage(lazySinks.dialog)
+    tooltips: TooltipsBackstage(lazySinks.dialog),
+    checkButtonContext: (specContext: string) => {
+      const resolveContext = () => {
+        if (Strings.contains(specContext, ':')) {
+          return specContext.split(':');
+        } else {
+          return [ specContext ];
+        }
+      };
```

Comment:
```suggestion
      const resolveContext = () => Strings.contains(specContext, ':'))
        ? specContext.split(':')
        : [ specContext ];
```

Reply:
Not sure you need a if or a ternary splitting a string with `:` that doesn't have a `:` will produce an array with just the original item.